src/components/Landing/Register.tsx

```diff
@@ -0,0 +1,19 @@
+const Register = () => {
+  //const { useAuth } = useAuth();
+  // const [user, setUser] = useState("");
+  // eslint-disable-next-line no-constant-condition
+  return 1 == 1 ? (
```

Comment:
what is this? 😂

I guess you're trying to do feature flag. there is some good blog here: https://www.somkiat.cc/feature-flag-driven-development/

my suggestion is that you can define when this feature being enabled using environment variable.